scripts/globals/events/login_campaign_data.lua

```diff
@@ -0,0 +1,463 @@
+-- This file is called by login_campaign.lua and require()'s no file, it should not be require()'d by any
+-- other lua scripts, which should instead require() login_campaign.lua directly.
+
+-- items table can only hold up to 20 items per option.
+tpz.events.loginCampaign.prizes =
```

Comment:
Do these IDs shift monthly, or are they part of a larger set determined by the client?

Reply:
Yes the prizes shift monthly. Here's a link to the monthly prices.
https://www.bg-wiki.com/ffxi/Repeat_Login_Campaign/Past_Login_Campaign_Rewards#Repeat_Login_Campaign_No._1

Reply:
I'm more thinking about the actual vendor options (or mask).  With RoE, there's rotating objectives, but the client just moves to a different mask/value.  For the items that change, is the selection the same?

Reply:
the mask changes when you buy an item and then go to a different price. Everytime that you buy an item the mask changes, this is already being handled in login_campaign =)